nautobot/circuits/tests/test_api.py

```diff
@@ -212,11 +214,18 @@ def setUpTestData(cls):
             Circuit.objects.create(cid="Circuit 2", provider=provider, type=circuit_type),
             Circuit.objects.create(cid="Circuit 3", provider=provider, type=circuit_type),
         )
+        cls.circuits = circuits
 
         CircuitTermination.objects.create(circuit=circuits[0], site=sites[0], term_side=SIDE_A)
         CircuitTermination.objects.create(circuit=circuits[0], site=sites[1], term_side=SIDE_Z)
         CircuitTermination.objects.create(circuit=circuits[1], site=sites[0], term_side=SIDE_A)
         CircuitTermination.objects.create(circuit=circuits[1], site=sites[1], term_side=SIDE_Z)
+        CircuitTermination.objects.create(
+            circuit=circuits[2], site=sites[0], term_side=CircuitTerminationSideChoices.SIDE_A
+        )
+        CircuitTermination.objects.create(
+            circuit=circuits[2], site=sites[0], term_side=CircuitTerminationSideChoices.SIDE_Z
+        )
```

Comment:
These two might be duplicated with cls.create_data below. They have the same `circuit` and same `term_side`